src/parser/mod.rs

```diff
@@ -4759,7 +4759,14 @@ impl<'a> Parser<'a> {
             Token::HexStringLiteral(ref s) => Ok(Value::HexStringLiteral(s.to_string())),
             Token::Placeholder(ref s) => Ok(Value::Placeholder(s.to_string())),
             tok @ Token::Colon | tok @ Token::AtSign => {
-                let ident = self.parse_identifier()?;
+                // Not calling self.parse_identifier()? because only in placeholder we want to check numbers as idfentifies
```

Comment:
✅  makes sense to me 

```suggestion
                // Not calling self.parse_identifier()? because only in placeholder we want to check numbers as identifies
```